To address the issue identified in SolidQueue where exceptions raised within the SolidQueue context are not properly identified, the following change should be implemented:

**Instruction:**

Modify the `wrap_in_app_executor` method located in `lib/solid_queue/app_executor.rb` as follows:

Replace the line:
```ruby
SolidQueue.app_executor.wrap(&block)
```

With:
```ruby
SolidQueue.app_executor.wrap(source: "application.solid_queue", &block)
```

**Purpose:**  
This modification ensures that the source of the error is correctly specified as "application.solid_queue", allowing for proper identification of exceptions originating within SolidQueue. This aligns with the guidance in the Rails error reporting documentation.